test/integration/metrics/metrics_test.go

```diff
@@ -241,3 +241,255 @@ func TestApiserverMetricsLabels(t *testing.T) {
 		}
 	}
 }
+
+func TestApiserverMetricsPods(t *testing.T) {
+	callOrDie := func(_ interface{}, err error) {
+		if err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+	}
+
+	makePod := func(labelValue string) *v1.Pod {
+		return &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   "foo",
+				Labels: map[string]string{"foo": labelValue},
+			},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{
+					{
+						Name:  "container",
+						Image: "image",
+					},
+				},
+			},
+		}
+	}
+
+	_, s, closeFn := framework.RunAMaster(framework.NewMasterConfig())
+	defer closeFn()
+
+	client, err := clientset.NewForConfig(&restclient.Config{Host: s.URL, QPS: -1})
+	if err != nil {
+		t.Fatalf("apiserver_request_total metric not exposed")
```

Comment:
Here should be
```
t.Fatalf("Error in create clientset: %v", err)
```
as the same as the existing test TestApiserverMetricsLabels.

Reply:
fixed! thanks